medcat/utils/meta_cat/ml_utils.py

```diff
@@ -271,6 +272,14 @@ def eval_model(model: nn.Module, data: List, config: ConfigMetaCAT, tokenizer: T
     predictions = np.argmax(np.concatenate(all_logits, axis=0), axis=1)
     precision, recall, f1, support = precision_recall_fscore_support(y_eval, predictions, average=score_average)
 
+    labels = [name for (name, index) in sorted(config.general['category_value2id'].items(), key=lambda x:x[1])]
+    confusion = pd.DataFrame(
+        data= confusion_matrix(y_eval, predictions,),
```

Comment:
hi @jamesbrandreth  - linter is failing due to the extra space here: `data= ` should be `data=confusion_matr..`